src/App.tsx

```diff
@@ -107,7 +112,7 @@ export const App: React.FC = () => {
           >
             <div className="tile is-child box is-success ">
               {selectedPost && (
-                <PostDetails post={selectedPost} />
+                <PostDetails />
```

Comment:
if we render this component only when selectedPost exists, I suggest you just pass it as a prop